Edit column names in bigquery table.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
+This is a step-by-step guide to help you change the name of a column in bigquery tables.
+
+## Step 1: Creating table with new schema in bigquery
+---
+In the **explorer** in bigquery in the dataset. Click on the ```⋮```to the right of the dataset name. And select the item **Create table**.  
+![create table](https://user-images.githubusercontent.com/38936255/146343440-b3db5c50-19b9-4314-9c7e-f82ca012d870.png)  
+And create a new table using the schema with the changed name of the field or fields.  
```

Comment:
And=and

Reply:
Create this more clean: https://github.com/NebraLtd/hm-diag/blob/0fe933265d2af405f6ae204e470fff3469f716f1/Edit%20column%20names%20in%20bigquery%20table.md?plain=1#L11-L14